superset/connectors/base/models.py

```diff
@@ -377,7 +402,7 @@ def get_column(self, column_name: Optional[str]) -> Optional["BaseColumn"]:
 
     @staticmethod
     def get_fk_many_from_list(
-        object_list: List[Any], fkmany: List[Column], fkmany_class: Type, key_attr: str,
+        object_list: List[Any], fkmany: List[Column], fkmany_class: Type, key_attr: str
```

Comment:
hmm, why'd we lose the trailing comma here? Are you on the right version of black?

Reply:
Let's avoid digging into styling changes that pass CI.